packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx

```diff
@@ -88,8 +88,10 @@ export class ExtendedSelectedItem extends BaseComponent<ISelectedPeopleItemProps
       </div >);
   }
 
-  private onClickIconButton = (action: (() => void) | undefined): () => void => {
-    return (): void => {
+  private onClickIconButton = (action: (() => void) | undefined): (ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void => {
```

Comment:
Why not just make this a method? Also private names should start with _

Reply:
I'll change the function name. 

I'm not sure I follow the first part of your comment. Do you mean make separate methods for the onRemoveItem and onExpandItem?

Reply:
Right, I'll change the function name.

I'm not sure I follow the first part of your comment. Do you mean make separate methods for the onRemoveItem and onExpandItem?